service-api/src/main/java/greencity/dto/notification/NotificationScheduleDto.java

```diff
@@ -1,14 +1,22 @@
 package greencity.dto.notification;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
-import lombok.RequiredArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
 import lombok.experimental.Accessors;
 
 import javax.validation.constraints.NotEmpty;
 
+@Builder
 @Data
 @Accessors(chain = true)
-@RequiredArgsConstructor
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
```

Comment:
@Data bundles the features of @ToString and @Getter / @Setter so you should remove this annotations